### PR TITLE
A few small fixes to the Readme

### PR DESCRIPTION
**Pull request by josephsofaer**
I think this reads a little better.
Great work @liangg !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,25 +10,25 @@ We have implemented connection pooling and other interesting features.
...
 
 ### Monitoring
 
 Airbnb MaxScale implements minutely internal metrics collection and exposes them for real time monitoring. A new `stats` command is added in MaxAdmin tool. External stats agent can be created to pull stats from a running MaxScale server periodically.
```
Comment (josephsofaer) [inline README.md:25]: @liangg I didn't understand what "minutely internal metrics collection" meant. Can you clarify that part?

Comment (tuckerman) [reply README.md:25]: nit: generated a MySQL query statement _which_ was

maybe consider s/completely removed/effectively removed

Comment (josephsofaer) [reply README.md:25]: @tuckerman updated

**Review by tuckerman — APPROVED** [review]
LGTM

**Pull request merged**